pkg/configuration/constants.go

```diff
@@ -18,6 +18,7 @@ const (
 	DEBUG_FORMAT                    string = "debug_format"
 	INSECURE_HTTPS                  string = "insecure"
 	PROXY_AUTHENTICATION_MECHANISM  string = "proxy_auth"
+	TEMP_DIR_PATH                   string = "snyk_tmp_dir_path"
```

Comment:
Suggestion: `snyk_` will translate into an environment variable as per naming convention. considering this, we might want to look at consistency with other examples. `SNYK_CACHE_PATH` for example, this would mean `SNYK_TMP_PATH` makes more sense.